### PR TITLE
Tutorial update

### PR DESCRIPTION
**Pull request by camilaViquez**
## Description
Reorganize some sections and add more descriptive steps so we can share this as a tutorial

### CODE REVIEW

File: Sama_Databricks_Connector_Tutorial.ipynb [hunk 1]
```diff
@@ -189,125 +193,106 @@
...
-    }
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
-    "from sama.constants import TaskStates\n",
-    "# returns a dataframe of task data and its current status. See docs for additional filters.\n",
-    "df = client.get_multi_task_status_to_table(spark, PROJECT_ID, omit_answers=False, state=TaskStates.NEW) \n",
-    "display(df)"
+    "#CREATE a DataFrame using this modified data\n",
```
Comment (edchan7-sama) [inline Sama_Databricks_Connector_Tutorial.ipynb:224]: This code block seems to be duplicated

File: Sama_Databricks_Connector_Tutorial.ipynb [hunk 2]
```diff
@@ -31,7 +30,10 @@
     }
    },
    "source": [
-    "# Install Sama SDK and Databricks Connector"
+    "# Step 1: Requirements\n",
+    "**1.1 Installation requirements**\n",
+    "Before you begin, make sure you install PySpark and Sama\n",
+    "By runing the following cell you will download and install PySpark, Sama, and their dependencies."
```
Comment (edchan7-sama) [inline Sama_Databricks_Connector_Tutorial.ipynb:36]: I think PySpark is installed by default in Databricks? Can you confirm?

**Review by edchan7-sama — COMMENTED** [review]
Looks great! Thanks for the edits. Just a few comments above.

Also, in the Python SDK tutorial, the "Sama SDK" font looks a bit off? Or are the words look misaligned by a pixel? Maybe squished?

File: Sama_Databricks_Connector_Tutorial.ipynb (hunk 2 above)
Comment (camilaViquez) [reply Sama_Databricks_Connector_Tutorial.ipynb:36]: Updated

File: Sama_Databricks_Connector_Tutorial.ipynb (hunk 1 above)
Comment (camilaViquez) [reply Sama_Databricks_Connector_Tutorial.ipynb:224]: Updated

**Pull request merged**